docs/cookbook/languages/rust.md

```diff
@@ -127,6 +127,16 @@ Popular projects used by the Nix community for this purpose are:
 [fenix]: https://github.com/nix-community/fenix
 [rust-overlay]: https://github.com/oxalica/rust-overlay
 
+An example flake is provided at [zmitchell/rust-toolchains][custom-toolchains].
```

Comment:
Do we want to callout flakes specifically in our docs other than as nix escape hatches?

Reply:
I think it makes sense when it's the only way to do something. This is one of those cases where it's the only way to do it.